### PR TITLE
Feat adding find and decrypt

### PR DESCRIPTION
**Pull request by grundyoso**
Adds support for finding issued passes and decrypting NFC payloads

### CODE REVIEW

File: src/main/java/com/passninja/model/Pass.java [hunk 1]
```diff
@@ -98,6 +98,18 @@ public static PassninjaResponse<Pass> get(String passType, String serialNumber)
         return request(RequestMethod.GET, RESOURCE + "/" + passType + "/" + serialNumber, null, Pass.class, null);
     }
 
+    public static PassninjaResponse<Pass> find(String passType) throws ApiException, IOException,
```
Comment (inginiir) [inline src/main/java/com/passninja/model/Pass.java]: looks like this method returns a single pass, am I right?

Comment (grundyoso) [reply src/main/java/com/passninja/model/Pass.java]: it should return all the passes issued for that passType... are you not seeing that?

Comment (grundyoso) [reply src/main/java/com/passninja/model/Pass.java]: do you know how to make this method return an array of `Pass`es ?

Comment (inginiir) [reply src/main/java/com/passninja/model/Pass.java]: Hi Richard! Yes, I know, if you grant me permissions I could push the required changes.

Comment (grundyoso) [reply src/main/java/com/passninja/model/Pass.java]: @inginiir sent you repo invite just now... let me know if you don't see it:

https://github.com/flomio/passninja-java/invitations

**Comment by grundyoso** [comment]
Closing in favor of https://github.com/flomio/passninja-java/pull/4

**Pull request closed without merging**